shared-libs/transitions/src/schedule/due_tasks.js

```diff
@@ -11,6 +11,8 @@ const lineage = require('@medic/lineage')(Promise, db.medic);
 const messageUtils = require('@medic/message-utils');
 
 const BATCH_SIZE = 1000;
+const clearFailing = () => config.get('clear_failing_schedules') || false;
+
```

Comment:
I am thinking may be we should default it to true now. We don't want years of messages to be on the scheduled messages past due date by default. What's your thought @dianabarsan  ?